lib/getAmazonImage.ts

```diff
@@ -1,4 +1,12 @@
-export const getAmazonImage = async (title: string) => {
+// Copyright (c) Gridiron Survivor.
+// Licensed under the MIT License.
+
+/**
+ * Amazon Image
+ * @param {string} title - The title of the search item
+ * @returns {object} - The image url and response status
```

Comment:
refactor (blocking): the function is async so even though three return value in the code looks like a plain object, it's a promise. See the function return type, `Promise<{ imageUrl: string | null, success: boolean }>`.

```suggestion
 * @returns {Promise<{ imageUrl: string | null, success: boolean }>
} - The image url and response status
```